src/data/showCaseData.js

```diff
@@ -0,0 +1,14 @@
+export const showCaseData=
+{"header":"More than 20,000 companies are using our service to hire their employees.",
+"companies":[ 
+    "facebook",
+    "amazon",
+    "microsoft",
+    "apple",
+    "google",
+    "airbnb",
+    "spotify",
+    "yahoo",
+    "netflix"
+]   
+}
```

Comment:
Great work on separating the data from the component and putting it into its own file!